app/models/school.rb

```diff
@@ -42,4 +49,45 @@ def in_england?
   def identifier
     "School-#{urn}"
   end
+
+  def tsf_eligible_establishment?
+    tsf_eligible_establishment_type_codes.include?(establishment_type_code)
+  end
+
+private
+
+  def tsf_eligible_establishment_type_codes
```

Comment:
Would love to move this into the DB so we can let Jake manage it via the admin console.